PhysicsTools/NanoAOD/python/dimuons_cff.py

```diff
@@ -0,0 +1,42 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.common_cff import *
+
+
+DiMuonVertex = cms.EDProducer("DiMuonVertexProducer",
+    src = cms.InputTag("slimmedMuons"),
+    beamSpot = cms.InputTag("offlineBeamSpot"),
+    vertices = cms.InputTag("offlineSlimmedPrimaryVertices"),
+    mu1Selection = cms.string ("pt>5 && abs(eta)<2.4"),
+    mu2Selection = cms.string ("pt>3 && abs(eta)<2.4"),
+    preVtxSelection = cms.string ("pt>5 && abs(eta)<2.4"),
+    postVtxSelection = cms.string ("userFloat('prob')>0.01"), # WP 0.01
+    RefitTracks = cms.bool (True),
+    FitTunePTransientTracks = cms.bool(True),
+)
+
+DiMuonTable = cms.EDProducer("SimpleCompositeCandidateFlatTableProducer",
+    src = cms.InputTag("DiMuonVertex","diMuons"),
+    cut = cms.string(""), #we should not filter on cross linked collections
+    name = cms.string("DiMuon"),
+    doc  = cms.string("dimuons that create a good vertex"),
+    singleton = cms.bool(False), # the number of entries is variable
+    extension = cms.bool(False), # this is the main table for the muons
+    variables = cms.PSet(
+      CandVars,
+      fit_pt = Var("userFloat('fit_pt')",float,doc="fitted pT of dimuon system",precision=12),
+      fit_eta = Var("userFloat('fit_eta')",float,doc="fitted eta of dimuon system",precision=12),     
+      fit_phi = Var("userFloat('fit_phi')",float,doc="fitted phi of dimuon system",precision=12),
+      fit_mass = Var("userFloat('fit_mass')",float,doc="fitted mass of dimuon system",precision=-1),
```

Comment:
please adapt precision to a sensible value and check that the rounding of the other variables is also adequate